eth/filters/api.go

```diff
@@ -559,6 +562,9 @@ func (args *FilterCriteria) UnmarshalJSON(data []byte) error {
 
 			case []interface{}:
 				// or case e.g. [null, "topic0", "topic1"]
+				if len(topic) > maxTopics {
```

Comment:
it should be a list of values as the topic candidate. Does it make sense to hard-cap it as 4?